src/supertux/command_line_arguments.cpp

```diff
@@ -355,6 +356,17 @@ CommandLineArguments::parse_args(int argc, char** argv)
         repository_url = argv[++i];
       }
     }
+    else if (arg == "--open-in-editor")
```

Comment:
Use `--edit-level` to keep current style (e.g. `--disable-sound`, `--play-demo`).